src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const selectedTab = id => {
+    if (id !== activeTabId) {
+      onTabSelected(id);
+    }
+  };
+
+  const active =
+    tabs.find(el => el.id === activeTabId) === undefined
+      ? tabs[0].id
+      : activeTabId;
+
+  const { content } = tabs.find(el => el.id === active);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li className={tab.id === active && 'is-active'} data-cy="Tab">
```

Comment:
The className assignment for the active tab should ensure a string is always returned. Consider using a ternary operator: `className={tab.id === active ? 'is-active' : ''}` to avoid potential issues with falsy values.